custom/conf/app.ini.sample

```diff
@@ -74,6 +74,34 @@ WORK_IN_PROGRESS_PREFIXES=WIP:,[WIP]
 ; List of reasons why a Pull Request or Issue can be locked
 LOCK_REASONS=Too heated,Off-topic,Resolved,Spam
 
+[repository.signing]
+; GPG key to use to sign commits, Defaults to the default - that is the value of git config --get user.signingkey
```

Comment:
Maybe add a comment that this uses `RUN_USER`'s git config.

Reply:
Done